pysoa/test/stub_service.py

```diff
@@ -378,7 +379,10 @@ def __enter__(self):
         mock_action = self._MockAction(name='{}.{}'.format(self.service, self.action))
 
         if self.body or self.errors:
-            mock_action.side_effect = lambda _: ActionResponse(self.action, errors=self.errors, body=self.body)
+            mock_action.return_value = ActionResponse(self.action, errors=self.errors, body=self.body)
+
+        if self.side_effect:
```

Comment:
This needs to be `elif`.

Reply:
you should be able to define both, `mock` deals with it internally, useful when you want the callable to do something interesting, but at the same time you want to set a `return_value`

Reply:
Hmmm. Okay. I can live with that. But I think the documentation should explicitly state that it's not recommended (can be confusing to people who later come and edit tests using this feature) and should only be used if you really know what you're doing.